Earthfile

```diff
@@ -419,7 +419,7 @@ dind:
 
 dind-alpine:
     FROM docker:dind
-    RUN apk add --update --no-cache docker-compose
+    RUN apk add --update --no-cache docker-compose jq
```

Comment:
Could we use alpine + the docker-auto-install.sh script, just like `dind-ubuntu` target below?

It would help with minimizing differences.

Reply:
I don't see why not -- the `docker:dind` image will already have `dockerd` installed, so that portion of the code should be skipped. It's only the ubuntu version which performs an `apt-get remove -y docker docker-engine docker.io containerd runc || true`, where as the alpine version doesn't do any of that.

I'll switch it over.